server/test/module-index-test.ts

```diff
@@ -0,0 +1,67 @@
+import ModuleIndex, { ModuleType } from '../src/module-index';
+
+import { expect } from 'chai';
+
+// 14
+describe('module-index', function() {
+  it('indexes all module types', async function() {
+    let workspaceRoot = `${__dirname}/fixtures/modules/all-modules`;
+
+    const index = new ModuleIndex(workspaceRoot);
+    await index.indexModules();
+
+    expect(index.getModules()).to.have.length(14);
```

Comment:
`expect(...).to.have.length(...)` is deprecated in favor of `expect(...).to.have.lengthOf(...)`